src/TraceEvent/Stacks/Stacks.cs

```diff
@@ -409,13 +409,15 @@ public class StackSourceSample
         /// </summary>
         public int Scenario { get; set; }
 
+        public int CpuNumber { get; set; }
```

Comment:
We are using this to expose the CPU # where a sample was taken in WPA. Need for graphing. In mind a sample stack would also have a CPU #, would it not - we just haven't exposed it in the past?

Reply:
Change from approach of modifying base StackSourceSample to using derived class LinuxPerfScriptStackSourceSample adding CpuNumber. This is because per brianrob not every StackSourceSample has a CPU #